src/gradients.ts

```diff
@@ -51,7 +51,7 @@ export function convertLinearGradient(
 	const { x1, x2, y1, y2 } = positionsForOrientation(orientation)
 
 	const getColorStops = (colorStop: gradientParser.ColorStop, index: number): SVGStopElement => {
-		const offset = `${(index / (colorStops.length - 1)) * 100}%`
+	        const offset = `${colorStop.length.value}${colorStop.length.type}`;
```

Comment:
I don't think this is correct SVG, according to MDN the `offset` attribute accepts either a number or a percentage. This would be valid only when `colorStop.length.type` is `%`, but it could be `px` or `em` which would not be valid to be added to the `offset` attribute.

Reply:
You are right, I tried to fix it in next commit. Thank you.